srcpkgs/musikcube/template

```diff
@@ -51,3 +47,11 @@ musikcube-sndio_package() {
 		vmove usr/lib/musikcube/plugins/libsndioout.so
 	}
 }
+
+musikcube-pipewire_package() {
+	depends="musikcube-${version}_${revision}"
```

Comment:
```suggestion
	depends="musikcube>=${version}_${revision}"
```

Reply:
you want this on the `sndio` package too?

Reply:
Yes, please :)

Reply:
done it already :p